src/Components/Projects/Projects.tsx

```diff
@@ -30,107 +43,130 @@ const handleClose = () => setModalState("close");
     navigate(`/dashboard/edit-pro/${id}`)
 
   }
+  const viewPro = (id)=>{
+    navigate(`/dashboard/view-pro/${id}`)
+  }
+    {/* show  Delete Modal */ }
   const showDeleteModel = (id)=>{
 
     setModalState("delete-modal")
     setItemId(id)
  
   }
-  const viewPro = (id)=>{
-    navigate(`/dashboard/view-pro/${id}`)
-  }
+    {/*  Delete Function */ }
   const DeleteProject =()=>{
     axios.delete(`${baseUrl}/project/${itemId}`,{headers:reqHeaders}).then((response)=>{
-      console.log(response);
+      getToastValue("success","DeletedSuccessfully!")
       handleClose()
-      showAllProjects()
+      showAllProjects();
     }).catch((error)=>{
-      console.log(error);
+      getToastValue('error',error?.response?.data?.message)
       
     })
   }
+
+  {/*  Filtration Function */ }
+  const getTitleValue =(title)=>{
+    setSearchInput(title.target.value);
+    showAllProjects(1,title.target.value)
+   }
+
   useEffect(() => {
-    showAllProjects()
+    showAllProjects();
   }, [])
   
   return (
     <>
-     <Modal show={modalState==='delete-modal'} onHide={handleClose}>   
-       <Modal.Body>
-       <div className="delete-container">
-          <div className="icons text-end">
-        <i onClick={handleClose} className="fa-regular fa-circle-xmark text-danger "></i>
-          </div>
-          <div className="text-center">
+      <Modal show={modalState === 'delete-modal'} onHide={handleClose}>
+        <Modal.Body>
+          <div className="delete-container">
+            <div className="icons text-end">
+              <i onClick={handleClose} className="fa-regular fa-circle-xmark text-danger "></i>
```

Comment:
The icon should be inserted inside a button, and the onClick should be given to the button instead for better user accessibility.